relay-dynamic-config/src/defaults.rs

```diff
@@ -117,6 +138,38 @@ pub fn add_span_metrics(project_config: &mut ProjectConfig) {
                 })
                 .into(),
         },
+        // Resource-specific tags:
+        TagMapping {
+            metrics: vec![
+                LazyGlob::new("d:spans/http.response_content_length@none".into()),
+                LazyGlob::new("d:spans/http.decoded_response_body_length@none".into()),
+                LazyGlob::new("d:spans/http.response_transfer_size@none".into()),
+            ],
+            tags: [
+                ("", "environment"),
+                ("span.", "description"),
```

Comment:
@DominikB2014 this was not mentioned in the Notion doc, but I assume you need the span description and group hash on these metrics as well?

Reply:
Yes we will, good call on that!